unitOneAssessmentPractice.js

```diff
@@ -91,6 +183,17 @@ let assert = require('assert')
 // Write a function called onlyLongStrings that removes all strings with 3 or fewer characters
 // Your function should use a higher-ordered function (e.g map, filter, reduce, every, sort) in its implementation
 // Your output should be in the same order as the input
+let longyArr = [];
+function onlyLongStrings(arr) {
```

Comment:
Filter is also another neat way of solving this: 
function onlyLongStrings(arr) {
  return arr.filter(el => el.length > 3
)
}